test/indentation-test-RRR.R

```diff
@@ -0,0 +1,349 @@
+
```

Comment:
I think these files should have -test- in their names. They are not tests as such.  It would also be nice to use these files as references.  I would simply call them `indentation-RRR.R` or `style-RRR.R`. Maybe even place them in a separate subfolder `samples/` or `styles/` and keep only the base name `RRR.R`.